### ISSUE TITLE
`rome lint --review` applies autofixes automatically (without user prompt)

### ISSUE THREAD

**Issue by hzoo** [facebookexperimental/rome#360]
## Description

Looks like `rome lint --review` applies autofixes automatically, similar to prettifying/formating. ([video](https://imgur.com/AOfzncM))

> Tested on latest commit: https://github.com/facebookexperimental/rome/commit/baaac53eb91bf360a777b3060819e0242359ae47

## Steps To Reproduce

1. Create a file with an autofixable rule (lint/preferTemplate)

```js
// `src.js`: 
'a' + 'b'
```

2. Run rome:

```sh
❯ rome lint --review
✔ Nothing to review!
```

## Expected Results

Unlike `rome lint --save`, prompt for autofixes when using `--review! Sebastian mentioned it was previously intentional but better to apply formatting only.

**Comment by sebmck** [comment]
Fixed by #361

**Comment by sebmck** [comment]
Thanks for the report! Just merged #361.
